src/systems/user_commands/UserCommands.cc

```diff
@@ -248,6 +332,24 @@ void UserCommands::Configure(const Entity &_entity,
       &UserCommandsPrivate::PoseService, this->dataPtr.get());
 
   ignmsg << "Pose service on [" << poseService << "]" << std::endl;
+
+  // Enable collision service
+  std::string enableCollisionService{
+    "/world/" + worldName + "/enable_collision"};
```

Comment:
This is something new since this PR was opened. We should pass these service names through the `AsValidTopic` function like was done on #522.